frontend/src/app/modules/workspace/file-browser-section/file-browser-section.component.ts

```diff
@@ -152,7 +157,8 @@ export class FileBrowserSectionComponent implements OnInit {
 
     nodeSelect(evt: any): void {
         const nodeSelected: TreeNode = evt.node;
-        if (nodeSelected.data !== 'Folder') {
+        if (nodeSelected.type === TreeNodeType.file.toString()) {
+            debugger;
```

Comment:
no debugger, please

Reply:
there is a lot of time yet before morning, in my last commit I will use ctrl + f for finding all occurrences of "debugger")